gammapy/stats/poisson.py

```diff
@@ -413,6 +414,84 @@ def _significance_direct_on_off(n_on, n_off, alpha):
 
     return significance
 
+def Helene_ULs(excess, error, conf_level=95.45):
+    """Compute Upper limit based on Helene et al., NIM 212 (1983) 319
+
+    Parameters
+    ----------
+    excess : double
```

Comment:
Write always "float" instead of "double", that's what the type is called in Python.

Reply:
Sorry, a too fast copy/paste. Corrected